### ISSUE TITLE
Defining to core product

### ISSUE THREAD

**Issue by aepstein** [aepstein/vumt#19]
The purpose of this issue is to capture discussion about what constitutes a minimum viable product for this project. (See: https://en.wikipedia.org/wiki/Minimum_viable_product)

The core function is simple: provide a comprehensive electronic replacement for the current paper based registration process at the trailhead.

- Users sign up and register online with the location they are headed toward similar to how they do at the trailhead now.  Except they are encouraged to do this before they leave for their trip.
- App asks users to specify some additional info not presently collected: where they expect to depart from, what kind of activity they are doing (e.g. paddle, snowshoe, hike, climb, etc.).
- When users arrive at the trailhead, they tap a button on the app to "sign in".
- The app figures out when they depart (using data on location, time, etc.) and prompts them to confirm.

**Comment by craigmcg** [comment]
MVP should support 4 distinct personas:
1. End user: hiker, paddler, etc
2. Ranger
3. Forester or other use planning employee
4. System administrator

**Comment by craigmcg** [comment]
I believe the core scenario above should also include a checkout function.  I'm not sure if it should be in the MVP to check-in before driving to the trailhead, although I think this is a nice feature.

**Comment by craigmcg** [comment]
If there is a permitting system in place, it should seamlessly interact with this system, including any printing that may be required.

**Comment by aepstein** [comment]
> If there is a permitting system in place, it should seamlessly interact with this system, including any printing that may be required.

Adding permitting potentially makes the application quite a bit more complex, depending on the details of the rules and the frequency with which they change.

**Comment by craigmcg** [comment]
Agree, but I wanted to capture the idea in case it became a critical feature.
